### DISCUSSION TITLE
iris an  face descriptors

### DISCUSSION THREAD

**Discussion by rickmaz** [vladmandic/human#469] [General]
Can I get the iris output and store it so that if the user comes back in the future, I can match his current iris scan with what I have in my database? Or can I save the entire image from a human and use face descriptors to store numerical uniqueness in the database for authentication later on?

**Comment by vladmandic** [comment]
that's what face descriptors are for. there are examples in demo folders.

**Comment by rickmaz** [comment]
do you know how many descriptors it uses? i know some models are 128 others
can use up to 512 for descriptor points? or is there an option to select
how many i want to store?

**Reply by vladmandic** [reply]
there are multiple models supported, each with different complexity. it's documented.
